content/tools/_index.md

```diff
@@ -2,14 +2,26 @@
 title: Tools
 ---
 
-We maintain a collection of tools, small and large, that help projects across the ecosystem:
+We maintain a collection of tools, small and large, that help projects across the ecosystem.
+All tools:
+
+- are released under the [3-Clause BSD](https://opensource.org/license/bsd-3-clause/) or [MIT](https://opensource.org/license/mit/) License;
+- assign copyright to `Scientific Python Developers`;
```

Comment:
Thoughts?

Reply:
I am not sure what the correct wording is or what we should say, but we've been doing it in practice. Not sure if we've done it consistently.

Reply:
transfer?

Reply:
Some of the projects will be started here (e.g, `lazy_loader`, `spin`, `yaml2ics`, etc.), so I am not sure about `transfer`. And some might be shared, so I am not sure `assign` is correct. But `share` doesn't sound correct when it just goes to `Scientific Python Developers`.

Here is a selection of some current examples:

- https://github.com/scientific-python/lazy_loader/blob/main/LICENSE.md
  Copyright (c) 2022--2023, Scientific Python project All rights reserved.
- https://github.com/scientific-python/pytest-doctestplus/blob/main/LICENSE.rst
  Copyright (c) 2023, Scientific Python Developers Copyright (c) 2011-2023, Astropy Developers
-  https://github.com/scientific-python/repo-review/blob/main/LICENSE
   Copyright (c) 2022, Henry Schreiner.
- https://github.com/scientific-python/scientific-python-hugo-theme/blob/main/LICENSE
  Copyright (c) 2021--2023, Scientific Python Developers
  Adapted from the Hugo Fresh theme, which has the following license:
  MIT License
  Copyright (c) 2019 Stefan M.

Reply:
I don't have strong feelings here but "attribute" seems perhaps to be the word with the least amount of additional connotation.

Reply:
Changed to `attribute`.